docs/integrations/destinations/README.md

```diff
@@ -1,5 +1,6 @@
-# Destinations
+---
+description: Data warehouses and databases
```

Comment:
Data warehouses, Data lakes, Databases...

Reply:
Ok made the change